akka-javasdk-testkit/src/main/java/akka/javasdk/testkit/EntitySerializationChecker.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (C) 2021-2024 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package akka.javasdk.testkit;
+
+import akka.javasdk.impl.serialization.JsonSerializer;
+import akka.runtime.sdk.spi.BytesPayload;
+
+public class EntitySerializationChecker {
```

Comment:
final and private constructor since not intended for extension or instantiation, or should it be package private since only intended to be used by other things in the testkit and not directly by the users.